### ISSUE TITLE
Cannot upload v1.2.0 to PyPI due to specific jwst_gtvt dependency

### ISSUE THREAD

**Issue by bourque** [ExoCTK/exoctk#497]
Using `twine upload -u '<username>' -p '<password>' --repository-url https://upload.pypi.org/legacy/ --skip-existing dist/*` we see:

```
Uploading exoctk-1.2.0-py3-none-any.whl
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 18.7M/18.7M [00:29<00:00, 664kB/s]
NOTE: Try --verbose to see response content.
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
Invalid value for requires_dist. Error: Can't have direct dependency: 'jwst-gtvt @ git+https://github.com/spacetelescope/jwst_gtvt.git@cd6bc76f66f478eafbcc71834d3e735c73e03ed5'
```

**Comment by hover2pi** [comment]
Hmmm, we still only have 1.1.0 on PyPI.

**Comment by hover2pi** [comment]
Resolved in #577
